packages/utils/src/color/color.ts

```diff
@@ -0,0 +1,90 @@
+import { colord, extend } from 'colord';
+import cmykPlugin from 'colord/plugins/cmyk';
+import namesPlugin from 'colord/plugins/names';
+
+import type { AnyColor } from 'colord';
+import type { Color } from '@pixi/utils';
```

Comment:
Could you please not import `@pixi/utils` from itself? It can create build warnings for us unnecessarily. Better to import relatively.